doc/user-getting-started-docker-dotnet.md

```diff
@@ -115,53 +100,29 @@ myst-appbuilder Dockerfile
 `/home`, `/bin`, etc. It also contains our application under `/app`.
 The dotnet runtime is also included.
 
-## Package the application
-
-dotnet runtime requires more heap memory than Mystikos provides
-by default. To expand the memory, we need to sign or package the application
-with a configuration file, in which a higher limit of heap size can be
-specified as follows:
-```
-{
-    // OpenEnclave specific settings
-    "Debug": 1,
-    "StackMemSize": "400k",
-    "NumUserThreads": 8,
-    "ProductID": 1,
-    "SecurityVersion": 1,
-
-    // Mystikos specific settings
-    // The heap size of the kernel and user application. Increase this setting
-    // if your app experienced OOM.
-    "MemorySize": "512m",
-    // The path to the entry point application in rootfs
-    "ApplicationPath": "/usr/bin/dotnet",
-    // The parameters to the entry point application
-    "ApplicationParameters": ["/app/sum.dll"],
-    // Whether we allow "ApplicationParameters" to be overridden by command line options of "myst exec"
-    "HostApplicationParameters": true,
-}
-```
-You can ignore most of the settings for now except for `MemorySize`. 512 MB is the minimum required
-by dotnet runtime, and you can increase it as needed. For details of the config file, please refer to
-[signing and packaging](./sign-package.md)
+## Create a CPIO archive and run the program inside a SGX enclave
```

Comment:
```suggestion
## Create a CPIO archive and run the program inside an SGX enclave
```

Reply:
Thanks for the feedback. Never expected `SGX` starts with a vowel. Changed all instances under Mystikos.